### PR TITLE
Cobblestone to Block of Rock

### PR DESCRIPTION
**Pull request by Tzabarpho**
This worked of course

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -3,6 +3,12 @@ plugins {
 	id 'maven-publish'
 }
 
+repositories {
+	flatDir {
+		dirs 'libs'
+	}
```
Comment (YTG1234) [inline build.gradle:9]: Nice repo btw

File: src/main/java/com/osomod/osomod2/mixin/AbstractBlockMixin.java [hunk 2]
```diff
@@ -32,6 +33,11 @@ public void onUse(BlockState state, World world, BlockPos pos, PlayerEntity play
                 world.setBlockState(pos, Blocks.GRANITE.getDefaultState());
             }
             cir.setReturnValue(ActionResult.SUCCESS);
+        } else if (state.isOf(Blocks.COBBLESTONE)) {
+            if (!world.isClient()) {
+                world.setBlockState(pos, CommonMain.ROCK.getDefaultState());
```
Comment (YTG1234) [inline src/main/java/com/osomod/osomod2/mixin/AbstractBlockMixin.java:38]: Makes sense I guess

File: src/main/resources/fabric.mod.json [hunk 3]
```diff
@@ -29,6 +29,7 @@
   ],
   "depends": {
     "fabricloader": ">=0.9.1",
-    "fabric": "*"
+    "fabric": "*",
+    "mymod": "*"
```
Comment (YTG1234) [inline src/main/resources/fabric.mod.json:33]: yes

**Review by YTG1234 — APPROVED** [review]
I think this thing is awesome and ready to go in the main mod

**Comment by Tzabarpho** [comment]
Thanks!

**Pull request merged**